packages/support/resources/views/components/actions/group-button.blade.php

```diff
@@ -0,0 +1,38 @@
+@props([
+    'actions',
+    'color' => null,
+    'darkMode' => false,
+    'icon' => 'heroicon-o-dots-vertical',
+    'label' => __('filament-support::actions/group.trigger.label'),
+    'size' => null,
+    'tooltip' => null,
+])
+
+<x-filament-support::dropdown
+    :dark-mode="$darkMode"
+    placement="bottom-end"
+    teleport
+    {{ $attributes }}
+>
+    <x-slot name="trigger">
+        <x-filament-support::button
+            :color="$color"
+            :dark-mode="$darkMode"
+            :icon="$icon"
+            :size="$size"
+            :tooltip="$tooltip"
+        >
+            <x-slot name="label">
+                {{ $label }}
+            </x-slot>
+        </x-filament-support::button>
+    </x-slot>
```

Comment:
Wouldn't it be easier to use the existing `packages/support/resources/views/components/actions/group` component and only swap out the trigger for button/link/icon button?

Reply:
I had done that, a parameter that I changed, but I was told to make
separate files because of the dark theme.

Reply:
@zepfietje do you have any suggestions on how to do it?  I had placed a Type property, and from there I changed the format, as I said @danharrin had commented about making separate files.

https://github.com/filamentphp/filament/pull/4905/commits/a8a8d1ff1a89293a213547dcc22fac4c2f9ff3f8

Reply:
@danharrin

Reply:
Sounds good. Shall we close this PR then, @danharrin?

Reply:
Leave it open and ill use it when i write this feature

Reply:
@danharrin When will version 3 come out, to count on this feature?

Reply:
@moskoweb, we don't know yet, also depends on Livewire v3.

Reply:
So this feature will be open until version 3, I will see to use my repository even then in the project, because I need this feature.

Reply:
You can use this in v2 without needing to use your fork

You can copy the new ActionGroup class and all the views into your app and use them like normal

Or keep the same ActionGroup that we use, copy the views into your app, and just pass ->view('your view') to the group when you need to change the button